crates/cheatcodes/src/env.rs

```diff
@@ -33,7 +33,7 @@ impl Cheatcode for envBool_0Call {
 impl Cheatcode for envUint_0Call {
     fn apply(&self, _state: &mut Cheatcodes) -> Result {
         let Self { name } = self;
-        env::<U256>(name, None)
+        env_uint(name).map_err(|e| fmt_err!("Could not parse U256 environment variable {name}. Make sure it's a valid uint, and ideally 0x-prefixed if its using hex: {e}"))
```

Comment:
this error message should be improved, the hint about 0x is redundant if we support non 0x prefix hex values?